dynamic_programming/longest_common_subsequence.dart

```diff
@@ -0,0 +1,62 @@
+import 'dart:math';
+import 'package:test/test.dart';
+
+// Prints the length of the longest common subsequence
+int longest_common_subsequence(String t1, String t2) {
+  int m = t1.length;
+  int n = t2.length;
```

Comment:
handle edge cases.

Reply:
I have handled all  edge cases now it's passing all test cases on leetcode.

![image](https://user-images.githubusercontent.com/64142943/196362558-8755c1f0-43cf-4c7f-9235-c666326af58d.png)